### PR TITLE
Generate "register-version" file automatically

### LINKED ISSUE #1207: Generate `register-version.js` file automatically
Instead if hardcoding the version in https://github.com/simplabs/ember-simple-auth/blob/1.2.0/vendor/ember-simple-auth/register-version.js we should probably generate that file automatically based on the information in the `package.json` file.

/cc @marcoow

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves #1207

### CODE REVIEW

**Comment by Turbo87** [comment]
@marcoow acceptance test might be hard, but a unit test that checks the registration is already in

**Comment by marcoow** [comment]
oh nice, wasn't aware even ;)

File: index.js [hunk 1]
```diff
@@ -28,5 +36,5 @@ module.exports = {
         app.import(asset, options);
       };
     }
-  }
+  },
```
Comment (marcoow) [inline index.js]: I thought we had a check in place that prevents extra trailing commas?

**Comment by Turbo87** [comment]
> oh nice, wasn't aware even ;)

got introduced by https://github.com/simplabs/ember-simple-auth/pull/1209

fixed the trailing comma issue

**Pull request merged**